desc/objectives/_geometry.py

```diff
@@ -656,9 +664,21 @@ def build(self, use_jit=True, verbose=1):
         if not np.allclose(plasma_grid.nodes[:, 0], 1):
             warnings.warn("Plasma grid includes interior points, should be rho=1")
 
+        # TODO: How to use with generalized toroidal angle?
+        errorif(
+            self._use_signed_distance
+            and not np.allclose(
```

Comment:
what is this checking?

Reply:
the signed distance algorithm I use only works if the points lie in the same toroidal plane, this is checking that. Check the link to the algorithm: https://doi.org/10.1016/B978-0-12-336156-1.50012-4